packages/headless/src/controllers/standalone-search-box/headless-standalone-search-box.ts

```diff
@@ -147,17 +134,17 @@ export function buildStandaloneSearchBox(
           enableQuerySyntax: options.enableQuerySyntax,
         })
       );
-      dispatch(
-        checkForRedirection({defaultRedirectionUrl: options.redirectionUrl})
-      );
+      dispatch(fetchRedirectUrl({id}));
     },
 
     get state() {
       const state = getState();
+      const standaloneSearchBoxState = state.standaloneSearchBoxSet[id]!;
       return {
         ...searchBox.state,
-        redirectTo: state.redirection.redirectTo,
-        analytics,
+        isLoading: standaloneSearchBoxState.isLoading,
```

Comment:
I made a small fix here. The `isLoading` property now reflects the status of the `/plan` request instead of the `/search` request.